packages/recommend/src/__tests__/getRecommendations.test.ts

```diff
@@ -125,4 +125,40 @@ describe('getRecommendations', () => {
       {}
     );
   });
+
+  test('override `undefined` threshold with default value', async () => {
```

Comment:
```suggestion
  test('overrides `undefined` threshold with default value', async () => {
```

Reply:
fixed in https://github.com/algolia/algoliasearch-client-javascript/pull/1300/commits/3007cb4ca213e0f85fc87c3218ee1a9caf101768